applications/Chat/evaluate/metric.py

```diff
@@ -0,0 +1,152 @@
+import jieba
+from nltk.translate.bleu_score import sentence_bleu
+from rouge_chinese import Rouge
+from bert_score import score
+from sklearn.metrics import precision_score, recall_score, f1_score
```

Comment:
Python imports should follow PEP-8 standards.

Please refer this guide to reorganise your imports: https://composed.blog/python/imports